cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
```

Comment:
I'm trying to think whether there are any implications, particularly on pegasus, for pages that aren't currently localized.  It's possible that seeing a non-en language cookie could cause them to display the wrong thing.  @tanyaparker do you happen to remember offhand?